static/scss/course-cards.scss

```diff
@@ -0,0 +1,105 @@
+// sass-lint:disable mixins-before-declarations
+.course-section h1 {
+  font-size: 44px;
+  font-weight: bold;
+  letter-spacing: 0;
+  line-height: 54px;
+  text-align: center;
+  padding-top: 54px;
+  padding-bottom: 54px;
+  margin: 0 auto;
+  max-width: 760px;
+
+  @include media-breakpoint-down(sm) {
+    font-size: 30px;
+    line-height: 40px;
+    padding-top: 20px;
+    padding-bottom: 22px;
+  }
+}
+
+.course-listing {
+  padding: 0;
+  margin: 0 -22px;
```

Comment:
This PR introduces a space at the right side of the page in the mobile view, and I think it is a result of this negative margin. Can you look into this?

![ss 2021-08-06 at 15 44 18 ](https://user-images.githubusercontent.com/14932219/128569572-3cb7e5f2-a051-4da6-b598-12f1735752ab.png)

Reply:
space seems same spacing on both side. I did not notice the difference. 

![screencapture-localhost-8013-homepage-2021-08-09-14_30_42](https://user-images.githubusercontent.com/7334669/128686045-6fb40a66-84fc-48c3-a42e-d2237e21b2b0.png)

Reply:
@asadiqbal08 I figured out what was happening. For some reason, when that negative margin is applied, I'm able to scroll very slightly to the right in the mobile view, and that's what causes the slight space to the right of the header and the main body of the page. I don't think it's likely that users will see this, so I'm fine leaving it as it is for now, and revisiting later if we see any issues during testing